api/Document.json

```diff
@@ -4313,6 +4399,92 @@
           }
         }
       },
+      "fullscreenerror_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/fullscreenerror_event",
+          "description": "<code>fullscreenerror</code> event",
+          "support": {
+            "chrome": {
+              "version_added": "45"
+            },
+            "chrome_android": {
+              "version_added": "45"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "64"
+              },
+              {
+                "version_added": "47",
+                "version_removed": "65",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "full-screen-api.unprefix.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "10",
+                "version_removed": "65",
+                "alternative_name": "mozfullscreenerror"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "64"
+              },
+              {
+                "version_added": "47",
+                "version_removed": "65",
```

Comment:
Does https://github.com/mdn/browser-compat-data/pull/3996#discussion_r278681832 apply to this? (I'm not sure, but it seems like it might.)